teps/0050-ignore-task-failures.md

```diff
@@ -187,6 +194,222 @@ control over the task definitions but may desire to ignore a failure and continu
 
   ![Jenkins Dashboard](images/0050-jenkins-dashboard-with-failure-stage.png)
 
+## Proposal
+We propose a new filed ```OnError``` to the [PipelineTask](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#adding-tasks-to-the-pipeline) definition.
+
+```go
+type PipelineTask struct {
+	Name string `json:"name,omitempty"`
+
+	//...
+
+	// OnError defines the termination behavior of a pipeline when the task is on error
+	// can be set to [ continue | stopAndFail ]
+	OnError OnErrorType `json:onError, "omitempty"`
```

Comment:
wonder if `OnFail` is a better name 🤔

Reply:
Good point, ```OnFail``` describes the behaviour better. It also helps developers/users distinguish ```pipelinTask.OnFail``` and ```step.OnError```, as they are different thing semantically. 

Updated to ```OnFail```

Reply:
I prefer `onError` to be consistent with the step level failure strategies - sorry @jerop 🤗 

But if others agree to go this route, I can approve it. The only downside with this name is - it might confuse users to understand the difference between two fields (`onError` vs `onFail`) given that the strategies are very similar.

Reply:
Putted it back to ```onError```, please take a look 😄